osquery/logger/logger.cpp

```diff
@@ -369,36 +358,11 @@ BufferedLogSink& BufferedLogSink::get() {
 }
 
 void BufferedLogSink::setUp() {
-  WriteLock lock(enable_mutex_);
```

Comment:
No need for mutex,  as we are not using active_ anymore.

 BufferedLogSink is setUp is called only in one place, so no need for additional checks.

Reply:
What if someone will add one more place calling `BufferedLogSink::setUp()` ? I'd suggest to leave mutex lock here for the sake of safe.

Reply:
Thanks for comment. 

AddLogSink is thread safe. It is using lock to protect it's own structure.

https://github.com/google/glog/blob/master/src/logging.cc#L619